src/components/layout/Sidebar.tsx

```diff
@@ -31,46 +33,70 @@ const Sidebar: FC = () => {
 
   const disableMenuItem = pathname === HOME_PATH;
 
-  return (
-    <GraaspMainMenu>
-      <LinkMenuItem
-        icon={<BarChartIcon />}
-        text={t('TAB_GENERAL')}
-        to={buildItemPath(itemId)}
-        disabled={disableMenuItem}
-      />
-      <LinkMenuItem
-        to={buildUsersAnalyticsPath(itemId)}
-        disabled={disableMenuItem}
-        icon={<PersonIcon />}
-        text={t('TAB_USERS')}
-      />
+  const { data: memberships } = hooks.useItemMemberships(itemId);
+  const { data: currentMember } = hooks.useCurrentMember();
+
+  const userMemberships = memberships
+    ?.filter((m) => m.member.id === currentMember?.id)
+    .reduce((acc: PermissionLevel[], curr) => [...acc, curr.permission], []);
+
+  const memberPermissionOverItem =
+    userMemberships && PermissionLevelCompare.getHighest(userMemberships);
```

Comment:
We recently added `permission` within item, which corresponds to the highest permission the user has over the item. I think you can simply use that property ;)